azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeStartCallOptions.java

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling.models;
+
+import com.azure.android.communication.ui.calling.CallComposite;
+
+import java.util.List;
+
+/**
+ * Start Call options to start 1 to N call experience using {@link CallComposite}.
+ */
+public class CallCompositeStartCallOptions {
```

Comment:
StartCallOptions sounds a bit ambiguous, since "starting a call" is something we do frequently.

Maybe ParticipantsToCallOptions

and instead of "participants", "invitees".